pkg/utils/version_test.go

```diff
@@ -40,11 +40,19 @@ func TestSATokenAutoGenerated(t *testing.T) {
 			want:              false,
 		},
 		{
-			kubeServerVersion: "v1.24.6+k3s1",
```

Comment:
This is the version of a k3s release.

Reply:
Really? Do you mind running `kubectl version` in k3s cluster and print the output here?

Reply:
Actually, I removed this case because it failed.

Reply:
![image](https://user-images.githubusercontent.com/15814167/200223756-513cc6b1-17f6-4596-a7bd-d02e53a09d0e.png)

Reply:
```bash
$ k3s kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.3+k3s1", GitCommit:"f2585c1671b31b4b34bddbb3bf4e7d69662b0821", GitTreeState:"clean", BuildDate:"2022-10-25T19:59:38Z", GoVersion:"go1.19.2", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.7
```

Reply:
A little bit confused, if [https://github.com/k3s-io/kubernetes](https://github.com/k3s-io/kubernetes) only have tags named `v*-k3s1`, then where `v1.25.3+k3s1` come from?
Also, in your output, only have client version, what about the server version?

Reply:
Updated.